catalog/models.py

```diff
@@ -0,0 +1,82 @@
+from django.db import models
+from django.urls import reverse  
+from django.conf import settings
+from venv import create
+
+ADDRESS_CHOICES = (
+    ('B', 'Billing'),
+    ('S', 'Shipping'),
+)
+class Customer(object):
```

Comment:
django-admin cung cấp sẵn class User,
em tìm hiểu Proxy model hoặc custom thằng User để dùng nhé
https://docs.djangoproject.com/en/4.0/topics/auth/customizing/#custom-users-and-proxy-models